experimenter/experimenter/targeting/constants.py

```diff
@@ -2561,6 +2574,21 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TOU_NOT_ACCEPTED_ADS_ENABLED_MAC_OR_WIN = NimbusTargetingConfig(
+    name="TOU not accepted yet, ads enabled, Mac or Win",
+    slug="tou_not_accepted_ads_enabled_mac_win",
+    description=(
+        "Users who have not accepted the terms of use yet, "
+        "have not disabled ads, "
+        "and are on Mac or Windows"
+    ),
+    targeting=f"((os.isWindows || os.isMac) && !({ACCEPTED_TOU_IN_NIMBUS_EXPERIMENT} || {ACCEPTED_TOU_IN_ON_TRAIN_ROLLOUT} || {ACCEPTED_TOU_V1}) && !{ADS_DISABLED})"
+    """,
```

Comment:
Just executing `python constants.py` blows up starting with a problem around here - presumably it's the " followed by the """.  It's not entirely obvious to me what's intended here.